org.eclipse.lsp4xml/src/main/java/org/eclipse/lsp4xml/services/XMLCompletions.java

```diff
@@ -183,7 +189,7 @@ private CompletionList collectAutoCloseTagSuggestion(int tagCloseEnd, String tag
 		item.setLabel("</" + tag + ">");
 		item.setKind(CompletionItemKind.Property);
 		item.setFilterText("</" + tag + ">");
-		item.setTextEdit(new TextEdit(new Range(pos, pos), "$0</" + tag + ">"));
+		item.setTextEdit(new TextEdit(new Range(pos, pos), "$0</" + tag + ">")); //"\n\t$0\n</" + tag + ">"

```

Comment:
be careful, not all clients support snippets. You should open a ticket to not return $number placeholders if client capabilities do not support snippets